en/SettingsView/General.md

```diff
@@ -1,49 +1,110 @@
 
-![RTK GPS Settings](../../images/settings/settings_view_general_rtk_gps.jpg)
+The settings are:
+- **Survey-in accuracy:** The minimum position accuracy for the RTK Survey-in process to complete.
+- **Minimum observation duration:** The minimum time that will be taken for the RTK Survey-in process. 
 
 
-## Autoconnect to the following devices
+## AutoConnect to the following devices {#auto_connect}
 
-This section defines the set of devices to which *QGroundControl* will auto-connect. Options include: **Pixhawk**, **SiK Radio**, **PX4 Flow**, **Libre Pilot**, **UDP**, **RTK GPS**.
+This section defines the set of devices to which *QGroundControl* will auto-connect. 
 
 ![Device autoconnect settings](../../images/settings/settings_view_general_autoconnect_devices.jpg)
 
+Settings include:
+- **Pixhawk**: Autoconnect to Pixhawk-series device
+- **SiK Radio**:  Autoconnect to SiK (Telemetry) radio
+- **PX4 Flow**: Autoconnect to PX4Flow device
+- **Libre Pilot**: Autoconnect to Libre Pilot autopilot
+- **UDP**: Autoconnect to UDP
+- **RTK GPS**: Autoconnect to RTK GPS device
+- **NMEA GPS Device / Baudrate**: If detected, GPS information will be used for ground station location and follow me support.
 
-## Video / Video Recording
-
-The *Video* section is used to define the source and connection settings for video that will be displayed in *Fly View*. The *Video Recording* section is used to specify the file format and maximum allocated file storage for storing video.
+## Video {#video}
 
-> **Note** The values displayed in this setting depend on the video source. If no video source is specified then no other video or video recording settings will be displayed (below we see the settings when UDP source is selected).
+The *Video* section is used to define the source and connection settings for video that will be displayed in *Fly View*.
 
 ![Video settings](../../images/settings/settings_view_general_video_udp.jpg)
+
+> **Note** The values displayed in this setting depend on the video source. 
+  If no video source is specified then no other video or *video recording* settings will be displayed (above we see the settings when UDP source is selected).
+
+
+## Video Recording
+
+The *Video Recording* section is used to specify the file format and maximum allocated file storage for storing video. Videos are saved to a sub-directory ("Video") of the [Application Load/Save Path](#load_save_path).
+
+![Video - without auto deletion](../../images/settings/settings_view_general_video_recording.jpg)
+
+![Video - auto deletion](../../images/settings/settings_view_general_video_recording_auto_delete.jpg)
+
+The settings are:
+- **Auto-delete Files**: If checked, files are auto deleted when the specified amount of storage is used.
+- **Max Storage Usage**: Maximum video file storage before video files are auto deleted.
+- **Video file format**: File format for the saved video recording.
+
+
+## Brand Image
```

Comment:
This appears to just be an icon that replaces the autopilot icon in the top right corner 
- is that correct?
- Is this always displayed irrespective of connected autopilot?
- Is the idea that someone might configure QGC for their own autopilot and always connect to item of that type?

Anything else to be said about this? Seems odd that it can be reset and does not affect the app icon etc. I guess I should start with "how/when/why" is it supposed to be used?